microservices-connector/config/manager/gmc-manager.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       containers:
       - command:
         - /manager
-        image: zhlsunshine/gmcmanager:latest
+        image: zhlsunshine/gmcmanager:oneclick
```

Comment:
Should we use opea/gmcmanager:latest here? Please avoid to use a personal docker repo.

Reply:
fixed